### DISCUSSION TITLE
What does a teal organization look like?

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#968] [Research]
In the second renaissance pilot course, we discussed what 'teal organizations' might look like. A few examples were given, like wisdom, responsibility, and development of consciousness. In this discussion post we can continue on this brainstorm.

**Comment by matthewmccarthy11** [comment]
1- The evolution/individuation of the 'person' and the 'organization' are mutually compatible (mutual synthesis of individual and environment @asimong) 

2- Self-maintenance- the 'self-maintenance' or 'reproduction' of the organization is not strictly or primarily 'financial', but 'cultural'. It is there because people want to be there, it exists 'for itself' (happy to expand more on this point as I think it is important)

In these discussions, curious also to see what you guys think of the notion of 'symbiotic cognition' from Marc Slors, I think this can be helpful when considering these questions: https://www.tandfonline.com/doi/full/10.1080/09515089.2019.1679591#:~:text=Symbiotic%20cognition%20provides%20a%20different,abilities%20than%20does%20extended%20cognition.... "social institutions constitute cognitive processes and abilities through task-dependency, more so than through functional integration. I will argue that characterizing cognition that is constituted by social institutions as symbiotic rather than distributed or extended cognition is more true to the nature of institutions and avoids the predicament of the previous section."

**Reply by asimong** [reply]
actually just realised that's a very slight misquote. "mutual synthesis of organism and environment" is the quote, the definition of health from the Peckham experiment. Thus healing can be seen as essentially enabling transformation — synthesis — and if you see transformation too clearly there is the danger of attaching to specific outcomes

**Reply by catherinet1** [reply]
I resonate with the suggested relationship between healing and transformation of "healing enables transformation" -- and also wonder whether it might also be that "healing _is_ transformation".

Also think it's an important point that attaching too rigidly to specific outcomes of what transformation looks like seems dangerous. How we can hold that _and_ stand firmly for values/principles we have faith in?

**Comment by rufuspollock** [comment]
Laloux's Reinventing Organizations (2014) lists about a dozen and provides detailed case studies.

Laloux was the one who (sort-of) coined the term "teal" organizations - the foreword of his book is written by Ken Wilber, and the intro chapter explicitly lays out a simple spiral dynamics-based evolutionary model of culture and human organization.

Detailed notes here: https://lifeitself.org/blog/reinventing-organizations-by-laloux-notes-summary

NB: Laloux perhaps inevitably ends up mostly describing governance and visible practices. The actual lived day to day culture of organizations is  less clear.

Also noticeable is that the two larger companies he documented "reverted" from their teal management practices (a software company and a power company).

**Reply by rufuspollock** [reply]
NB: I would also now suggest that Laloux mainly describes (mature) green organizations rather than teal ones -- and we have little idea what teal organizations actually look like yet.

**Reply by asimong** [reply]
I'm appreciating your nuance here, @rufuspollock — on the other hand, if Laloux is part of the movement **defining** teal, then one can hardly say that he isn't describing teal organisations, but maybe more that our definitions of teal vary. So, rather than relying on the colour label, let's be clearer as to what we mean. I don't mean necessarily a strict logical definition of the different Wilber (et al) stages (though I'd say that Kegan is better than most at offering a definition in terms of subject-object theory) but any clear enough description that makes sense and can be agreed on.

**Reply by rufuspollock** [reply]
Makes me think we should gather definitions and examples from these various places (eg Kegan stages, teal orgs, teal in wilber etc) and then contrast and compare - this would be useful in its own right.

Eg

- Kegan
- Laloux
- Wilber
- Beck

I also think there is a lot of confusion with personal  (ego) development and cultural stages.

**Reply by asimong** [reply]
> Detailed notes here: https://lifeitself.org/blog/reinventing-organizations-by-laloux-notes-summary

Wow, very detailed indeed!

**Comment by asimong** [comment]
Wnat I sense as an emergent question / issue here is: how do we balance up between, 3 way: 
1. (often individual) intuitive feelings of where something belongs, what "level" etc.
2. collective coming together in ontological commoning, achieving a common sense of the levels (under whatever name)
3. something measurable and more objective, and potentially amenable to scientific methodology

I've seen rather too many conversations where it boils down to "I see it this way" and "I see it that way". Yes, nice and "green" … everyone entitled to their own opinion … but where do we common the sensemaking? And if we don't common the sensemaking, where are we left in our collective search for communication; mutual understanding; collaborative action; mutual recognition and valuing?

The "I see it this way" vs "I see it that way" I also see replicated in open source forking. Guys! Stop forking! It's nice and green, but falls into all the traps of not seeing beyond the green.

**Comment by asimong** [comment]
To echo [C.E.M. Joad](https://en.wikipedia.org/wiki/C._E._M._Joad), "It all depends on what you mean by…" Teal. Maybe, @matthewmccarthy11, you're inadvertently sneaking in the question "what is Teal" in here, without asking it explilcitly? How about setting out what **you** mean by Teal, then we can actually give sensible opinions... As, personally, I don't think "teal" is well-enough defined for me to be happy to give a definition.

**Comment by asimong** [comment]
Or, you might also try reframing the question a bit more like: "Many contemporary organisations seem to conform to Spiral Dynamics 'green' patterns [and you'll need to list those patterns, with examples]. What would be the next stage of organisational evolution, characterised by [x, y, and z: characteristics of teal, or yellow, or 5th order, or deliberately developmental, or whatever]"

**Comment by matthewmccarthy11** [comment]
Good points, @asimong, and I wouldn't say I had a precise angle when asking the question- and was more so relaying what was brought up in the class. But I do think it is an interesting question, and not one I think I have an answer to. The interesting thing for me is: how does 'teal' make us rethink what an organization is? What you mentioned about the 'inside'/'exterior' distinction is an important (maybe the most important) part. And I think considering 'what is a teal organization'? is an opportunity to rethink how we view an organization as entangled by the people in it, because teal is more sensitive to 'relational dynamics'. And so, I guess the real question here is: what is an organization? Curious to read more about what said about collective ikiagi, and going to reemphasize here about 'symbiotic cognition' as an interesting angle.

**Reply by asimong** [reply]
Understood, so you were indeed fishing, @matthewmccarthy11 !

One of the things I find most interesting at this stage of a new enquiry is, just, can we define a question or set of questions that we can at least roughly agree on the meaning or significance of? The point being, that until we agree on the signification, we won't be having a conversation that connects.

**Reply by matthewmccarthy11** [reply]
Yeah- good point. Although I think that this can still serve as the overarching question: 'what is a teal organization/what does it look like'? 

I would just say that the angle I am approaching it from is that it makes one reconsider 'what is an organization?' 
Another sub question might be: how have people defined teal organizations? Or, how can one better clarify 'teal' from other related concepts like personal development? Or, are there any examples of teal organizations? Another sub question, or point of inquiry, to what degree are definitions merely 'selected', and who gets to choose? 

Lots of different questions/subs questions, but I do think the overarching question can house these other ones and serve as the 'main question'

**Reply by asimong** [reply]
My persistent question then is, well, what are we doing throwing an ill-defined term into the ring? Maybe (following @rufuspollock ) we are just taking Laloux's examples as the examples that define the concept of teal... in which case, are we just asking, what exactly are the common characteristics shared by those organizations and absent from others?

But my sense here (correct me if I'm mistaken) is that you maybe looking to sidestep Laloux — in which case, it would be most helpful (for you? for us?) either to comment on his examples, or to provide other ones, or posit some characteristics yourself, or write a sci-fi piece narrating what life would be like. With the usual hazards of fictional (utopian) writing....
